pyonepassword/_py_op_cli.py

```diff
@@ -4,6 +4,7 @@
 import logging
 from json.decoder import JSONDecodeError
 import subprocess
+import pexpect
```

Comment:
need to:
- add this to `setup.py`
- restrict to posix: `install_requires = ["pexpect; os_name==posix"]`
- Conditionally import `pexpect` so it doesn't blow up on windows
- Raise an exception if MFA code is provided on windows